services/QuillLMS/client/app/bundles/Teacher/components/dashboard/teacher_info_modal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react'
+
+import GradeLevelSection from '../accounts/shared/grade_level_section'
+import SubjectAreaSection from '../accounts/shared/subject_area_section'
+import { requestPost, } from '../../../../modules/request/index'
+
+const teacherAtBoardSrc = `${process.env.CDN_URL}/images/onboarding/packs-whole.svg`
+
+const TeacherInfoModal = ({ close, subjectAreas, }) => {
+  const [minimumGradeLevel, setMinimumGradeLevel] = React.useState(null)
+  const [maximumGradeLevel, setMaximumGradeLevel] = React.useState(null)
+  const [selectedSubjectAreaIds, setSelectedSubjectAreaIds] = React.useState([])
+
+  function submitTeacherInfo() {
+    requestPost(
+      `${process.env.DEFAULT_URL}/teacher_infos`,
+      {
+        minimum_grade_level: minimumGradeLevel.value,
+        maximum_grade_level: maximumGradeLevel.value,
+        subject_area_ids: selectedSubjectAreaIds
+      },
+      (body) => {
+        close()
+      }
+    )
+  }
+
+  function skipForNow() {
+    requestPost(
+      `${process.env.DEFAULT_URL}/milestones/create_or_touch_dismiss_teacher_info_modal`,
+      {},
+      (body) => {
+        close()
+      }
+    )
+  }
+
+  const saveButtonClassName = "quill-button primary contained medium focus-on-light"
+  let saveButton = <button className={`${saveButtonClassName} disabled`} disabled type="button">Save</button>
+
+  if (selectedSubjectAreaIds.length && minimumGradeLevel && maximumGradeLevel) {
+    saveButton = <button className={saveButtonClassName} onClick={submitTeacherInfo} type="button">Save</button>
+  }
+
+  return (
+    <React.Fragment>
+      <div className="modal-background" />
+      <div className="teacher-info-modal">
+        <div className="modal-body">
+          <div className="account-form add-teacher-info">
+            <img alt="" src={teacherAtBoardSrc} />
```

Comment:
do we not want an `alt` label here?

Reply:
I left it out because it's [purely decorative](https://www.w3.org/WAI/tutorials/images/decorative/), so it just becomes auditory clutter.